packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1993,12 +2008,6 @@ export class Grapher
                 title: this.isPlaying ? `Pause` : `Play`,
                 category: "Timeline",
             },
-            {
-                combo: "f",
-                fn: (): void => this.toggleFacetControlVisibility(),
-                title: `Toggle Faceting`,
-                category: "Chart",
-            },
```

Comment:
why did you remove this shortcut?

Reply:
It existed previously as an escape hatch to reveal the faceting checkbox even on charts where the author had chosen to hide it. It wasn't a shortcut to change the faceting mode, it just showed/hid the facet checkbox that used to exist in the controls row

Reply:
If it's easy to do I think there is no harm in keeping it (I've been using it a lot when debugging faceting issues). But no worries if it's difficult to make work :)

Reply:
Makes sense, added it back